hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -66,7 +66,7 @@ def __init__(self) -> None:
         self.trace: List[TraceDmemStore] = []
         self.pending: Dict[int, int] = {}
 
-    def _load_5byte_le_words(self, data: bytes) -> None:
+    def _load_5byte_le_words(self, data: bytes, offset: int = 0) -> None:
```

Comment:
Also, please don't give this private function a default value for `offset`. The only call sites are in this file and they all provide the argument.